assembl/tasks/sudoer.py

```diff
@@ -0,0 +1,307 @@
+    c.sudo('apt-get update; apt-get install -y docker-ce')
+    gitpath = 'https://github.com/docker/compose/releases/download/1.22.0/docker-compose-$(uname -s)-$(uname -m)'
+    c.run('curl -L %s -o /usr/local/bin/docker-compose' % gitpath)
+    c.sudo('chmod +x /usr/local/bin/docker-compose')
+    add_user_to_group(c.context.user, 'docker')
+
+
+@task()
+def install_fail2ban(c):
+    """Install fail2ban"""
+    c.sudo('apt-get install -y fail2ban')
+
+
+@task()
+def install_jq(c):
+    """Install jq package. jq is like sed for json data."""
+    c.sudo('apt-get install -y jq')
+
+
+@task()
+def install_mysql(c):
+    """Installs mysql on a linux server."""
+    c.run("apt-get -y install mysql-server")
+
+
+@task()
+def install_apache(c):
+    """Installs apache on a linux server."""
+    c.sudo("apt-get install apache2")
+
+
+@task()
+def install_php(c):
+    """Installs php on a linux server."""
+    c.sudo("apt-get -y install php php-mysql php-curl php-cli php-gd")
+
+
+@task()
+def uninstall_lamp(c):
+    """
+    Installs Apache2, Mysql and PHP on a Linux Environment, for dev purposes
+    """
+    c.sudo("apt-get purge apache2 mysql-server php-mysql php-curl php-cli php-gd")
+    c.sudo("apt-get autoremove")
+
+
+@task()
+def set_fail2ban_configurations(c):
+    """Set fail2ban configuration and pushes fail2ban configs."""
+
+
+@task()
+def clear_aptitude_cache(c):
+    c.run('sudo apt-get autoclean -qq && sudo apt-get autoremove -qq && sudo rm -rf /var/lib/apt/lists/*')
+
+
+@task()
+def install_build_dependencies(c):
+    """Build the necessary packages in order for CI/CD machines to build an Assembl wheel"""
+    c.sudo('apt-get update -qq')
+    c.sudo('apt-get install -yqq %s' % ' '.join(core_dependencies))
+    c.run('pip install -r requirements-build.txt')
+
+
+@task()
+def install_node_and_yarn(c):
+    """Compiles the necessary packages in order for transpilers to build static asset code. It will often
+    be used in the CI/CD context.
+
+    This is different than updating node inside of a virtual environment, and nodeenv is not used. However,
+    the node version's MUST be mactching.
+    """
+    node_version = c.config.get('node_version', '10.13.0')
+    node_major_version = node_version.split('.')[0]
+    c.sudo('apt-get update -qq')
+    if is_integration_env(c):
+        c.sudo('apt-get remove -yqq cmdtest')
+    c.run('curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -')
+    c.run('echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list')
+    c.run('wget -qO- https://deb.nodesource.com/setup_%s.x | sudo bash -' % node_major_version)
+    c.sudo('apt-get install -yqq nodejs yarn')
+
+
+@task()
+def install_chrome_dependencies(c):
+    """
+    Installs the requirements for chrome + pupeteer to be run. Assumes to be run on Debian based machines
```

Comment:
should be infinitive, Install and Assume. see https://www.python.org/dev/peps/pep-0257/
pupeteer => puppeteer

Reply:
Cool. Done.